chispa/__init__.py

```diff
@@ -43,8 +46,14 @@
 
 
 class Chispa:
-    def __init__(self, formats=DefaultFormats(), default_output=None):
-        self.formats = formats
+    def __init__(self, formats: FormattingConfig | None = None, default_output=None):
+        if not formats:
+            self.formats = FormattingConfig()
+        elif type(formats) is FormattingConfig:
```

Comment:
We are using `isinstance` in some places and `type is` in other. Let's, maybe, use `isinstance` everywhere?